### PR TITLE
Fixes to get the dump working with snowflake

### PR DESCRIPTION
**Pull request by bderenzi**

### CODE REVIEW

File: export_to_disk.sql [hunk 1]
```diff
@@ -1 +1 @@
-\copy (SELECT ROW_TO_JSON(t) FROM (SELECT * FROM json_export) AS t) TO '/tmp/json_export.json' CSV QUOTE '$';
+\copy (SELECT json_doc FROM json_export) to /tmp/json_export.csv CSV QUOTE '"';
```
Comment (roboton) [inline export_to_disk.sql:1]: do we want to interpret the file as a CSV by other systems? or as a json file? (even though technically its a csv file with a single column that is json)

Comment (roboton) [reply export_to_disk.sql:1]: only concern with this change is its ingestion/signaling to other systems downstream on how to read this file

**Review by roboton — COMMENTED** [review]
Thanks for testing/debugging!

File: export_to_disk.sql (hunk 1 above)
Comment (bderenzi) [reply export_to_disk.sql:1]: Just to confirm: you're proposing we go back to a `.json` filetype?

Would it be confusing with the quotes or is that not too big of a deal?

It's a bit of a weird thing and I suppose more `json` than `csv`.

**Comment by roboton** [comment]
Oh it ended up with big quotes around the entire JSON? if so maybe we can
remove those and then save it as a json?

**Comment by bderenzi** [comment]
Correct. It might also do some escaping inside -- I didn't look closely.

If we remove the big quotes, it will break again because of the issue with #1. This big quote solves #1 and gives us something that snowflake can import (because Snowflake can only use the `'` or `"` characters to surround fields)

**Comment by roboton** [comment]
I understand the ambiguity now, could go either way but since it only
contains one column (and no comma delimiters) I would lean towards json or
txt over csv. An interpreter expecting a csv may go and decide to wreak
havoc using the internal quoted commas inside the string to delimit fields
if it's not being careful about quoting. On the Snowflake side is the input
specified with a specific type: csv or json or something else?

**Comment by bderenzi** [comment]
Snowflake doesn’t care about the file name. One does specify whether to use
a CSV or JSON parser. In this case it needs to use a CSV parser to handle
the quoting around the JSON field correctly

I’m happy to go with your favorite. As a tie breaker I might lean towards
txt since it’s not a valid JSON file but all good by me

**Comment by roboton** [comment]
Oh interesting, so Snowflake reads it as CSV and then knows that it should
parse the contents of the single column as a JSON?

**Comment by bderenzi** [comment]
Yes. Snowflake reads it as a CSV file with a single column. I import it
into a table with a single column of type VARIANT, which is the JSONP
equivalent. Once it’s stored in a VARIANT column we have access to all of
the unstructured data commands for passing/using the JSON

**Comment by roboton** [comment]
Thanks for the explanation, I can see why CSV made sense. I think we should leave it as CSV and file a lower priority issue to get the quoting issues fixed so that the file can be imported as a JSON file. This works for now though. LGTM!

**Comment by bderenzi** [comment]
Okay, super. Merging

**Pull request merged**